### PR TITLE
Exp: ban and unban

### PR DESCRIPTION
**Pull request by Arrays-Arr**

### CODE REVIEW

File: wavelength-plugins/exp.js [hunk 1]
```diff
@@ -208,17 +208,25 @@ exports.commands = {
...
-		this.sendReply("You are no longer exempt from exp");
+	expunban: function (target, room, user) {
+		if (!this.can('roomowner')) return false;
+		if (!target) return this.parse('/help expunban');
+		let targetId = toId(target);
+		if (!Db.expoff.has(targetId)) return this.errorReply(targetId + ' is not currently exp banned.');
+		Db.expoff.remove(targetId);
+		this.sendReply(targetId + " is no longer banned from exp");
 	},
+	expunbanhelp: ['/expunban target - allows a user to gain exp if theyy were exp banned'],
```
Comment (DeathlyPlays) [inline wavelength-plugins/exp.js]: they*

**Comment by DeathlyPlays** [comment]
Maybe you should also keep expoff and expon but make it for regular user's but they can only disable/enable their own.

Second thought: They can only enable it back if they weren't expbanned?

**Comment by Arrays-Arr** [comment]
I really see no reason for them to disable their own. It prevents them from earning rewards only thing that'd be useful for is bans

**Comment by DeathlyPlays** [comment]
~~Same reason we made it to begin with @Arrays-Arr kek~~

**Pull request merged**